src/modules/map/components/openlayers/OpenLayersWMSLayer.vue

```diff
@@ -51,35 +53,63 @@ export default {
         },
     },
     created() {
-        let tileGrid = undefined
+        let source = undefined
+        if (this.gutter !== -1) {
+            source = new TileWMS({
+                projection: this.projection,
+                url: this.url,
+                gutter: this.gutter,
+            })
+        } else {
+            source = new ImageWMS({
+                url: this.url,
+                projection: this.projection,
+            })
+        }
         // If we are using LV95, we can constrain the WMS to only request tiles over Switzerland
         if (this.projection === CoordinateSystems.LV95.epsg) {
-            tileGrid = new TileGrid({
+            const tileGridLV95 = new TileGrid({
                 resolutions: TILEGRID_RESOLUTIONS,
                 extent: TILEGRID_EXTENT,
                 origin: TILEGRID_ORIGIN,
                 tileSize: WMS_TILE_SIZE,
             })
+
+            source.tileGrid = tileGridLV95
+
+            // tile grid and  reprojection to WebMercator is done in analogy to WMTS to prevent
+            // that the layer appears twice, once in CH and once near New Zealand.
+            // see: https://github.com/geoadmin/web-mapviewer/commit/c689f9a650c546c6e52a91fc2086d7cbbf48faa2
+            if (this.gutter !== -1) {
```

Comment:
I am also not 100% sure if it is correct to do the reprojection only for the TileLayer and not in case of ImageLayer.
But for ImageLayer this did not seem to be possible and probably not necessary, if I correctly understood this.

Reply:
I think you are right on this, I checked OpenLayer's API documentation and ImageWMS class doesn't have a function called `setTileGridForProjection`. So if you do not filter for the gutter options you might end up with an error in the console (and no layer shown on the map...)